src/components/ProfileBox/ProfileBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './ProfileBox.module.scss';
+
+import Edit from '#src/icons/Edit';
+import Check from '#src/icons/Check';
+import IconButton from '#components/IconButton/IconButton';
+import defaultAvatar from '#src/assets/profiles/default_avatar.png';
+
+type Props = {
+  name?: string;
+  image: string;
+  adult?: boolean;
+  editMode?: boolean;
+  onClick: () => void;
+  onEdit: () => void;
+  selected?: boolean;
+};
+
+const ProfileBox = ({ name, image, editMode = false, onClick, onEdit, selected = false }: Props) => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={classNames(styles.inner, selected && styles.selected)}>
+        <div onClick={onClick} className={styles.box}>
```

Comment:
`role="button" tabIndex={0} onKeyDown={keyDownHandler}` is missing here for keyboard access (and a11y)